plivo.py

```diff
@@ -20,15 +22,28 @@ class PlivoError(Exception):
     pass
 
 
-def validate_signature(uri, post_params, signature, auth_token):
-    for k, v in sorted(post_params.items()):
+def validate_signature(uri, params, signature, auth_token):
+    qs = urlparse(uri).query
+    if qs:
+        params_from_qs = dict(parse_qsl(qs))
```

Comment:
> Does Plivo ever append their own query string params to a URL?

I just realized that's what happens when you set your application's Message/Answer/Fallback/Hangup Method to GET